test/rdf-components-mocha.js

```diff
@@ -0,0 +1,139 @@
+// rdf-components-mocha.js
+
```

Comment:
Good tests for the cases that all succeed.  Would be good to add some tests where things go wrong e.g., query or update fails, so we can verify error handling.  This does not block merging this down - it's a suggestion for future improvement next time we are in here.